keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -1086,6 +1086,24 @@ An operator can query for PersistentVolumeClaims and PersistentVolumes in the
 cluster with the ReadWriteOncePod access mode. If any exist then the feature is
 in use.
 
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [X] Other
+  - Details:
+    - Create a Pod using a PersistentVolumeClaim with the ReadWriteOncePod access mode
+    - A PersistentVolume should be created and the PersistentVolumeClaim .status.phase=Bound
```

Comment:
Shouldn't the case be more about the second pod?
A second pod pointing to the same PV will not be scheduled, and you should see that in metrics.

Reply:
Good point, this example should include two pods. Updated to outline the status conditions and events for both pods